project.clj

```diff
@@ -11,7 +11,7 @@
   :dependencies [[org.clojure/clojure "1.8.0"]
                  [org.clojure/java.jdbc "0.6.1"]
                  [org.postgresql/postgresql "9.4.1208.jre7"]
-                 [migratus "0.8.28"]
+                 [migratus "0.8.29" :exclusions [org.clojure/java.jdbc]]
```

Comment:
Why the exclusion?  I see the latest version of java.jdbc is 0.6.2-alpha3.  Are we concerned with updating java.jdbc to the most recent instead of excluding versions?   Note there aren't any other exclusions in this.